src/components/SearchDrawer/ResultsList.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable react/display-name */
+import Link from 'next/link';
+import { useIntl } from 'react-intl';
+import { useQuery } from 'react-query';
+import { FunctionComponent, useEffect, useState } from 'react';
+
+import Search from '@material-ui/icons/Search';
+import {
+    Avatar,
+    Button,
+    List,
+    ListItem,
+    ListItemAvatar,
+    ListItemText,
+    ListSubheader,
+} from '@material-ui/core';
+
+import getOrg from '../../fetching/getOrg';
+import { MINIMUM_CHARACTERS } from './index';
+import { ZetkinPerson } from '../../types/zetkin';
+
+
+interface ResultsListProps {
+    searchFieldValue: string;
+    results: ZetkinPerson[];
+    loading: boolean;
+    orgId: string;
+}
+
+const ResultsList: FunctionComponent<ResultsListProps> = ({ searchFieldValue, results, loading, orgId }): JSX.Element => {
+    const intl = useIntl();
+    const [numResultsToDisplay, setNumResultsToDisplay] = useState<number>(5);
+    const { data: org } = useQuery(['org', orgId], getOrg(orgId), { enabled: false });
+
+    // If results change, reset the max number back to 5
+    useEffect(() => {
+        setNumResultsToDisplay(5);
+    }, [results]);
+
+    return (
+        <List>
+            { /* Typing prompts */ }
+            { searchFieldValue.length === 0 && (
+                <ListItem>
+                    <ListItemText>
+                        { intl.formatMessage({
+                            id: 'layout.organize.search.startTypingPrompt',
+                        }) }
```

Comment:
I think you should use the `<FormattedMessage>` (often abbreviated `<Msg>`) component instead of `intl.formatMessage()`.

If you `import { FormattedMessage as Msg } from 'react-intl'` you could replace these three lines with:

```tsx
<Msg id="layout.organize.search.startTypingPrompt"/>
```

I'm of course open to abstaining from the abbreviated import. It's just something we started doing in the old Organize to reduce some typing in places where there are a lot of messages (which is true for most components).

Reply:
Done!